src/tree/SubscriptionTreeItem.ts

```diff
@@ -149,7 +149,8 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
             }
         }
     }
-    private async initPostgresChild(server: PostgresAbstractServer): Promise<AzureTreeItem> {
+    private async initPostgresChild(server: PostgresAbstractServer, context: IActionContext): Promise<AzureTreeItem> {
```

Comment:
I don't think you want to get telemetry for every time the children tree items load.  If you're trying to see how many people are creating these server types, I think this seems like where you'd want it:

https://github.com/microsoft/vscode-cosmosdb/blob/main/src/postgres/commands/createPostgresServer/steps/PostgresServerCreateStep.ts#L51

Reply:
I asked Neelima to add this and I think it's helpful for various reasons. For example, the code you linked doesn't cover people who created outside of VS Code. Secondly, knowing the serverType here will help us investigate any errors - we've already seen unique errors for flexible vs. single (https://github.com/microsoft/vscode-cosmosdb/pull/1890)